golang/tracereporter.go

```diff
@@ -64,7 +64,7 @@ func NewBinaryAnnotation(key string, value interface{}) BinaryAnnotation {
 
 // TraceReporter is the interface used to report Trace spans.
 type TraceReporter interface {
-	Report(span Span, annotations []Annotation, binaryAnnotations []BinaryAnnotation)
+	Report(span Span, annotations []Annotation, binaryAnnotations []BinaryAnnotation) (bool, *error)
```

Comment:
Document the return values. Name them (purely for documentation) and add a method doc to the interface with what the `bool` is.